src/scripts/main.js

```diff
@@ -1,7 +1,30 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.classList.add('notification');
+  message.classList.add(type);
```

Comment:
```suggestion
  message.className = `notification ${type}`;
```

Reply:
you can use this to avoid repeating .classList.add()
If you'll want to add 10 classes ?:)